### ISSUE TITLE
[Bug] Storage used is above 100%

### ISSUE THREAD

**Issue by KevinLiss** [MauriceNino/dashdot#1008]
### Description of the bug

I have an external USB Drive mounted once on the host system. In dash it is shown as "used 144%", what cannot be correct

![image](https://github.com/MauriceNino/dashdot/assets/14164023/dd022b2c-0276-49c7-9b8e-b8f5e19b9494)

### Relevant log output

```shell
df output, seems that my "/mnt/host/media/usb" is used twice?

overlay              113809604  35413444  72568784  33% /
tmpfs                    65536         0     65536   0% /dev
shm                      65536         0     65536   0% /dev/shm
/dev/sda1            113809604  35413444  72568784  33% /mnt/host
udev                   3986172         0   3986172   0% /mnt/host/dev
tmpfs                  4009412         0   4009412   0% /mnt/host/dev/shm
tmpfs                   801884      3736    798148   0% /mnt/host/run
tmpfs                     5120         0      5120   0% /mnt/host/run/lock
tmpfs                   801880         0    801880   0% /mnt/host/run/user/1000
systemd-1            960303848 707647016 203802320  78% /mnt/host/media/usb
/dev/sdb             960303848 707647016 203802320  78% /mnt/host/media/usb
```

### Info output of dashdot cli

```shell
yarn run v1.22.19
$ node dist/apps/cli/main.js info
INFO
=========
Yarn: 1.22.19
Node: v20.10.0
Dash: 5.5.0

Cwd: /app
Hash: 2ec1e0f46c942404f834a9da2e097c2f434e19a2
In Docker: true
In Podman: false
In Docker (env): true
```

### Where is your instance running?

Linux Server

**Comment by MauriceNino** [comment]
Hi there, can you please post the output of the following command?

```
docker exec <CONTAINER> yarn cli raw-data --storage
```

**Comment by KevinLiss** [comment]
----------------------------------------

<details>
  <summary>Output:</summary>

```js
const disks =  [
  {
    device: '/dev/sda',
    type: 'SSD',
    name: 'K2              ',
    vendor: 'ATA',
    size: 120034123776,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '0807',
    serialNum: '',
    interfaceType: 'SATA',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sdb',
    type: 'HD',
    name: 'USB 3.0 Destop H',
    vendor: 'ASMT',
    size: 1000204886016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '0',
    serialNum: '',
    interfaceType: 'USB',
    smartStatus: 'unknown',
    temperature: null
  }
]
const sizes =  [
  {
    fs: 'overlay',
    type: 'overlay',
    size: 116541034496,
    used: 39132753920,
    available: 71441047552,
    use: 35.39,
    mount: '/',
    rw: false
  },
  {
    fs: '/dev/sda1',
    type: 'ext4',
    size: 116541034496,
    used: 39132753920,
    available: 71441047552,
    use: 35.39,
    mount: '/mnt/host',
    rw: true
  },
  {
    fs: 'systemd-1',
    type: 'autofs',
    size: 983351140352,
    used: 745736282112,
    available: 187587837952,
    use: 79.9,
    mount: '/mnt/host/media/usb',
    rw: false
  },
  {
    fs: '/dev/sdb',
    type: 'ext4',
    size: 983351140352,
    used: 745736282112,
    available: 187587837952,
    use: 79.9,
    mount: '/mnt/host/media/usb',
    rw: true
  }
]
const blocks =  [
  {
    name: 'sda',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 120034123776,
    physical: 'SSD',
    uuid: '',
    label: '',
    model: 'K2',
    serial: '',
    removable: false,
    protocol: 'sata',
    group: '',
    device: '/dev/sda'
  },
  {
    name: 'sdb',
    type: 'disk',
    fsType: 'ext4',
    mount: '/mnt/host/media/usb',
    size: 1000204886016,
    physical: 'HDD',
    uuid: '34a8cfcd-35fd-4797-ac43-69c1f176762a',
    label: '',
    model: 'USB 3.0 Destop H',
    serial: '',
    removable: false,
    protocol: 'usb',
    group: '',
    device: '/dev/sdb'
  },
  {
    name: 'sda1',
    type: 'part',
    fsType: 'ext4',
    mount: '/mnt/host',
    size: 119009181696,
    physical: '',
    uuid: '3a3a6d61-726f-475f-a9b1-f028b3814021',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: '',
    device: '/dev/sda'
  },
  {
    name: 'sda2',
    type: 'part',
    fsType: '',
    mount: '',
    size: 1024,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: '',
    device: '/dev/sda'
  },
  {
    name: 'sda5',
    type: 'part',
    fsType: 'swap',
    mount: '[SWAP]',
    size: 1022361600,
    physical: '',
    uuid: 'a96465f5-3925-456b-bd02-a3590c6c8179',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: '',
    device: '/dev/sda'
  },
  {
    name: 'sr0',
    type: 'rom',
    fsType: '',
    mount: '',
    size: 1073741312,
    physical: 'CD/DVD',
    uuid: '',
    label: '',
    model: 'DVD-RW DU8AESH',
    serial: '',
    removable: true,
    protocol: 'usb',
    group: ''
  }
]
```

</details>
